Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -152,6 +152,7 @@ type Props = {
   onDialogCreateTrigger?: (projectId: string, dialogId: string) => void;
   onDialogDeleteTrigger?: (projectId: string, dialogId: string, index: number) => void;
   defaultSelected?: Partial<TreeLink>;
+  showDelete?: boolean;
```

Comment:
Nit: One nice thing to do in props once you have a bunch of flags for different things and if you have good defaults is to put them in an options structure.

options?: {
    showTriggers?: boolean,
    showDialogs?: boolean,
    showDelete?: boolean
}

Reply:
I like this, but another thing to consider is why there needs to be so many options. Is this component being used in too many contexts? Is there some decomposition that could happen?